tests/semanticdb/metac.expect

```diff
@@ -49,44 +49,47 @@ Schema => SemanticDB v4
 Uri => Advanced.scala
 Text => empty
 Language => Scala
-Symbols => 34 entries
-Occurrences => 90 entries
+Symbols => 37 entries
+Occurrences => 96 entries
 
 Symbols:
-advanced/C# => class C
-advanced/C#[T] => typeparam T
-advanced/C#`<init>`(). => primary ctor <init>
-advanced/C#t(). => method t
-advanced/Structural# => class Structural
-advanced/Structural#`<init>`(). => primary ctor <init>
-advanced/Structural#s1(). => method s1
-advanced/Structural#s2(). => method s2
-advanced/Structural#s3(). => method s3
-advanced/Test. => final object Test
-advanced/Test.e. => val method e
-advanced/Test.e1. => val method e1
-advanced/Test.e1x. => val method e1x
-advanced/Test.s. => val method s
-advanced/Test.s1. => val method s1
-advanced/Test.s1x. => val method s1x
-advanced/Test.s2. => val method s2
-advanced/Test.s2x. => val method s2x
-advanced/Test.s3. => val method s3
-advanced/Test.s3x. => val method s3x
-advanced/Wildcards# => class Wildcards
-advanced/Wildcards#`<init>`(). => primary ctor <init>
-advanced/Wildcards#e1(). => method e1
-local0 => abstract val method x
-local1 => abstract val method x
-local2 => final class $anon
-local3 => val method x
-local4 => abstract method m
-local5 => param x
-local6 => final class $anon
-local7 => method m
-local8 => param x
-local9 => val local e3
-local10 => val local e3x
+advanced/C# => class C [typeparam T ] extends Object { self: C[T] => +3 decls }
+advanced/C#[T] => typeparam T 
+advanced/C#`<init>`(). => primary ctor <init> [unknown T: <?>]: C[T]
```

Comment:
it seems the type parameters for constructors are not initialised correctly, these are part of a `Template` object so whatever is done for `DefDef` probably needs to be copied to the ctor of a `Template`

Reply:
Fixed by https://github.com/tanishiking/dotty/commit/a8a137c36977125b3d65e2c2289268961ce2db14